storybookjs/storybook
# Composition: Fix undefined/undefined in url on init

Issue: #11264 

## What I did
- ADD check for api/modules/stories, to only navigate to valid storyId + viewmode
- FIX spelling error in story_store
- ADD default values for viewMode & storyId to FramesRenderer

## How to test
- Start a storybook, have it auto-open in the browser
- expected: The first story is selected in the manager & preview shows it correctly
